bindings/nodejs/lib/client/client.ts

```diff
@@ -106,6 +117,74 @@ export class Client {
         return JSON.parse(response).payload;
     }
 
+    /**
+     * Check the readiness of the node to issue a new block, the reference mana cost based on the rate setter and
+     * current network congestion, and the block issuance credits of the requested account.
+     */
+    async getAccountCongestion(
+        accountId: AccountId,
```

Comment:
Ideally the function parameter names would be the same as in the TIP, right?

Reply:
If it makes more sense, but bech32 address could be also Ed25519 or something, which wouldn't work. That's why we have the accountId here

Reply:
yes, and I like that much more than in the TIP

Reply:
Can this be closed?